### PR TITLE
eliminate cross-namespace HPA escalation attack

### PR DESCRIPTION
**Pull request by deads2k**
Cherry-pick of https://github.com/kubernetes/kubernetes/pull/16668

### CODE REVIEW

File: docs/api-reference/extensions/v1beta1/definitions.html [hunk 1]
```diff
@@ -3923,6 +3923,7 @@ <h3 id="_v1beta1_subresourcereference">v1beta1.SubresourceReference</h3>
 <td class="tableblock halign-left valign-top"></td>
 </tr>
 <tr>
+<<<<<<< HEAD
```
Comment (deads2k) [inline docs/api-reference/extensions/v1beta1/definitions.html]: I see it, trying to regenerate.

**Comment by bgrant0607** [comment]
cc @brendandburns

**Comment by bgrant0607** [comment]
@alex-mohr This is the cherrypick to the 1.1 branch. We'd like this to go in before we cut the release candidate.

**Comment by alex-mohr** [comment]
@bgrant0607 Done

**Comment by bgrant0607** [comment]
Thanks

**Pull request merged**